siman/wrappers/calculations_wrap.py

```diff
@@ -0,0 +1,102 @@
+import os
```

Comment:
make a more specific name for this file, which shows what are this wrappers about